Makefile

```
@@ -0,0 +1,64 @@
+# IC Nix environment version
+IC_NIX_VERSION := latest
+
+.PHONY: build build-wasm generate-did clean help test setup-nix-env nix-shell-env
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)